biopandas/__init__.py

```diff
@@ -24,5 +24,5 @@
 # 'X.Y.dev0' is the canonical version of 'X.Y.dev'
 #
 
-__version__ = "0.4.2"
+__version__ = "0.5.0"
```

Comment:
Ah sorry to come back to this, but 

`__version__ = "0.5.0"` should probably be `__version__ = "0.5.0dev"` so that we can collect some of the other things in the PR, and then we can change it to 0.5.0 when the the next release version goes out

Reply:
Yep good point, I'll try to wrap the other two PRs tonight too :)